### ISSUE TITLE
Allow to resize card or slider

### ISSUE THREAD

**Issue by juanjol** [KartoffelToby/better-thermostat-ui-card#139]
Hi! This is a question or maybe a functionality request:

I really like the design of this card, but in the wallpanel I'm setting up it's too small. I've tried to increase the size of the slider by adjusting the styles with card-mod, but I haven't been able to. Is there any way to make this manageable, or any way to overwrite it? From what little I've seen in the code, the 233px width is fixed.

**Comment by ThomDietrich** [comment]
I agree, would be nice to be able to resize. Here is a comparison between the default thermostat card and this one:

![image](https://github.com/KartoffelToby/better-thermostat-ui-card/assets/2870104/41dc9ee3-9c4e-4ab9-a4c8-0992af18684d)
